src/index.js

```diff
@@ -30,9 +30,118 @@ export { default as SearchField } from './SearchField';
 export { default as StatefulButton } from './StatefulButton';
 export { default as StatusAlert } from './StatusAlert';
 export { default as Table } from './Table';
-export { default as Tabs } from './Tabs';
+export { default as Tabs } from './Tabs'; // TODO: shim from react-bootstrap
 export { default as TextArea } from './TextArea';
 export { default as ValidationFormGroup } from './ValidationFormGroup';
 export { default as TransitionReplace } from './TransitionReplace';
 export { default as ValidationMessage } from './ValidationMessage';
 export { default as Variant } from './utils/constants';
+
+// Pass through components from react-bootstrap
+
+/*
+  TODO: Accordion
+  - a11y defect: Trigger button doesn't show expanded/collapsed state. Issue logged some time ago.
+
+export { default as Accordion } from 'react-bootstrap/Accordion';
+export { default as AccordionContext } from 'react-bootstrap/AccordionContext';
+export { default as AccordionCollapse } from 'react-bootstrap/AccordionCollapse';
+export {
+  default as AccordionToggle,
+  useAccordionToggle,
+} from 'react-bootstrap/AccordionToggle';
+
+*/
+export { default as Alert } from 'react-bootstrap/Alert';
+export { default as Badge } from 'react-bootstrap/Badge';
+/*
+  TODO: Breadcrumb
+  a11y defect: aria-current (needed semantics) is on the <li>, not on the <a>, which takes focus
+
+export { default as Breadcrumb } from 'react-bootstrap/Breadcrumb';
+export { default as BreadcrumbItem } from 'react-bootstrap/BreadcrumbItem';
+*/
+// TODO: create shim – export { default as Button } from 'react-bootstrap/Button';
+export { default as ButtonGroup } from 'react-bootstrap/ButtonGroup';
```

Comment:
Are these Button* components compatible with our Button above, that we have yet to shim?

Reply:
They are! I tested them in the doc page just added.